### DISCUSSION TITLE
UI returns 404

### DISCUSSION THREAD

**Discussion by krokojoss** [gerbera/gerbera#1385] [Q&A] [answered]
Ubuntu 20.04.2 LTS and install with 
1. sudo apt-get install gerbera=1.6.0-1~ppa1  
2. sudo apt-get install gerbera (updates to 1.8.0)

sudo systemctl status gerbera 
● gerbera.service - Gerbera Media Server with MySQL
     Loaded: loaded (/lib/systemd/system/gerbera.service; enabled; vendor preset: enabled)
     Active: active (running) since Fri 2021-04-09 16:10:58 EEST; 2s ago
   Main PID: 19397 (gerbera)
      Tasks: 19 (limit: 19073)
     Memory: 10.5M
     CGroup: /system.slice/gerbera.service
             └─19397 /usr/bin/gerbera -c /etc/gerbera/config.xml

**Comment by whyman** [comment, marked as answer]
What does your log file (or journalctl) say?

**Reply by krokojoss** [reply]
Today the output is as below. Noticed now it says MySQL, the config file has<sqlite3 enabled="yes">  Previously I had a working 1.1.0 installation with SQLite. In fact MySQL is not even running on the computer. I could try to install it, or use SQlite if that is possible

-- Reboot --
apr 11 20:13:43  systemd[1]: Started Gerbera Media Server with MySQL.
apr 11 20:14:21  gerbera[1121]: 2021-04-11 20:14:21   info: Gerbera UPnP Server 1.8.0 - http://gerbera.io/
apr 11 20:14:21  gerbera[1121]: 2021-04-11 20:14:21   info: Copyright 2016-2021 Gerbera Contributors.
apr 11 20:14:21  gerbera[1121]: 2021-04-11 20:14:21   info: Gerbera is free software, covered by the GNU General Public License version 2
apr 11 20:14:21  gerbera[1121]: 2021-04-11 20:14:21   info: Loading configuration from: /etc/gerbera/config.xml
apr 11 20:14:21  gerbera[1121]: 2021-04-11 20:14:21   info: Checking configuration...
apr 11 20:14:22  gerbera[1121]: 2021-04-11 20:14:22   info: Configuration check succeeded.
apr 11 20:14:24  gerbera[1121]: 2021-04-11 20:14:24   info: About to migrate metadata from mt_cds_object to mt_metadata
apr 11 20:14:24  gerbera[1121]: 2021-04-11 20:14:24   info: No data will be removed from mt_cds_object
apr 11 20:14:24  gerbera[1121]: 2021-04-11 20:14:24   info: Migrated metadata - object count: 0
apr 11 20:14:24  gerbera[1121]: 2021-04-11 20:14:24   info: Loading 0 configuration items from database
apr 11 20:14:24  gerbera[1121]: 2021-04-11 20:14:24   info: Initialising libupnp with interface: <unset>, port: <unset>
apr 11 20:14:24  gerbera[1121]: 2021-04-11 20:14:24   info: IPv4: Server bound to: 192.168.1.156:49152
apr 11 20:14:24  gerbera[1121]: 2021-04-11 20:14:24   info: IPv6: Server bound to: fe80::35c1:fc66:f99:3e41:49152
apr 11 20:14:24  gerbera[1121]: 2021-04-11 20:14:24   info: IPv6 ULA/GLA: Server bound to: 2001:999:200:b57:d36a:ace9:5b9e:4e85:49153
apr 11 20:14:25  gerbera[1121]: 2021-04-11 20:14:25   info: The Web UI can be reached by following this link: http://192.168.1.156:49152/

**Reply by krokojoss** [reply]
Installed MariaDB, created the database, added a user, changed config.xml and restarted gerbera. Works!
Strange that the server did not report an error about the missing db server

**Reply by whyman** [reply]
You can change which DB is used in your `config.xml`.

**Reply by krokojoss** [reply]
the original setting was SQlite but Gerbera still tried MySql. I can now watch our old digitized VHS videos on the TV set. For some reason VLC on Android did not display up the DLNA directory. Anyway case closed and thank you
